autolink/postlink/gradleLinker.js

```diff
@@ -1,44 +1,76 @@
 // @ts-check
-var path = require('./path');
-var fs = require('fs');
-var { warnn, errorn, logn, infon, debugn } = require('./log');
-var { insertString } = require('./stringUtils');
-var DEFAULT_KOTLIN_VERSION = '1.3.61';
+var path = require("./path");
+var fs = require("fs");
+var { warnn, errorn, logn, infon, debugn } = require("./log");
+var { insertString } = require("./stringUtils");
+var DEFAULT_KOTLIN_VERSION = "1.3.61";
 // This should be the minSdkVersion required for RNN.
-var DEFAULT_MIN_SDK_VERSION = 19
+var DEFAULT_MIN_SDK_VERSION = 19;
 
 class GradleLinker {
   constructor() {
     this.gradlePath = path.rootGradle;
+    this.setKlotinVersionSuccess = false;
+    this.setKotlinPluginDependency = false;
+    this.setMinSdkVersion = false;
```

Comment:
To be consistent should we add the suffix `success` to the two properties above?